EM.Sparql

```diff
@@ -0,0 +1,313 @@
+
+Title: Unused Object Property Detection
+Constraint Description: Identifies object properties that are defined but not used in any instances.
+Severity: Warning
+
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+SELECT ?property
+WHERE {
+  ?property a owl:ObjectProperty .
+  FILTER NOT EXISTS {
+    ?subject ?property ?object .
+  }
+}
+
+​The aim of this query is to identify object properties within the ontology that are defined but not utilized in any instances.​ 
+Detecting such unused properties helps maintain a clean and efficient ontology by identifying potentially redundant definitions.
+
+4. Finding Classes with Conflicting Disjointness and Subclass Relationships (level 6 - 3 points)
+
+Title: Disjoint Subclass Conflict
+Constraint Description: Identifies classes that are marked as disjoint but share a common superclass.
+Severity: Error
+
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+SELECT ?class1 ?class2
+WHERE {
+  ?class1 owl:disjointWith ?class2 .
+  ?class1 rdfs:subClassOf ?commonSuperclass .
+  ?class2 rdfs:subClassOf ?commonSuperclass .
+}
+
+The goal of this query is to identify classes that are marked as disjoint but are simultaneously defined as subclasses of a common superclass.​ 
+This situation presents a logical inconsistency within the ontology, as disjoint classes should not share any common subclasses.
+
+5. Identifying Classes with Different Labeling Standards (level 6 - 3 points)
+
+Title: Inconsistent Class Labeling
+Constraint Description: Detects classes with different labeling standards.
+Severity: Warning
+
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+
+SELECT ?class1 ?class2 ?commonSuperclass
+WHERE {
+  # Find two disjoint classes
+  ?class1 owl:disjointWith ?class2 .
+  
+  # Identify their common superclass
+  ?class1 rdfs:subClassOf ?commonSuperclass .
+  ?class2 rdfs:subClassOf ?commonSuperclass .
+}
+
+The goal of this query is to find classes within an ontology that are associated with different labeling standards.​ 
+This can be important for identifying inconsistencies in how classes are labeled or described, which can help improve the quality and usability of the ontology.
+
+6. Title: Identifying Classes with Multiple Definitions in CCO (level 5 - 5 points)
+
+This query retrieves classes within the Common Core Ontology (CCO) that have two or more different definitions, indicating potential redundancies or overlaps in property definitions.
+
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?class ?property1 ?definition1 ?property2 ?definition2
+WHERE {
+  # Selecting all classes in CCO
+  ?class a owl:Class .
+
+  # Retrieve the first property-definition pair for the class
+  ?class ?property1 ?definition1 .
```

Comment:
You need to leverage the specific annotation property used in CCO for definitions to narrow the focus of your query. Everything with a ? preceding it is a variable and so by default ranges over everything that might match its position in the triple.